src/GameState/Board.tsx

```diff
@@ -1,48 +1,64 @@
-import { Stack, HStack, Box } from 'native-base';
+import { Box, View } from 'native-base';
 import React from 'react';
-import { BOARD_GREEN, BOARD_GREY } from './boardColors';
+import { ImageBackground } from 'react-native';
+import { BOARD_GREEN } from './boardColors';
+import { convertBoardToBoardCoordinate } from './convertBoardToBoardCoordinate';
 import { Marble } from './Marble';
 
 export const Board = (props: any) => {
   const board = props.board;
-  const boardSize = props.preview ? 4 : 12;
-  const marbleColorSize = props.preview ? 4 : 12;
-  const marbleEmptySize = props.preview ? 2 : 4;
+  const boardSize = props.preview ? '128px' : '364px';
+  const marbleColorSize = props.preview ? '16px' : '48px';
+  const marbleEmptySize = props.preview ? '8px' : '16px';
 
   const [marbleClickedCoordinates, setMarbleClickedCoordinates] =
     React.useState<{ x: number; y: number } | null>(null);
 
+  const boardCoordinate = convertBoardToBoardCoordinate(board);
+  const boxMarbleSize = 100 / board.length;
+  const boxMarbleSizePourcent = `${boxMarbleSize}%`;
+
   return (
-    <Stack rounded={'lg'}>
-      {board.map((row: [], rowIndex: number) => (
-        <HStack alignItems="center" key={rowIndex}>
-          {row.map((cell, cellIndex) => (
-            <Box
-              size={boardSize}
-              alignItems="center"
-              justifyContent="center"
-              key={cellIndex}
-              bg={
-                marbleClickedCoordinates?.x === cellIndex &&
-                marbleClickedCoordinates?.y === rowIndex
-                  ? BOARD_GREY
-                  : BOARD_GREEN
-              }
-              m={0}
-              p={0}
-            >
+    <View width={boardSize} height={boardSize}>
+      <ImageBackground
+        source={require('./board.png')}
```

Comment:
Maybe import it ?

Reply:
This modification has been realized in the following PR